tutorial/westeros/westeros.ipynb

```diff
@@ -0,0 +1,1268 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "beta = 0.7 # an elasticity of demand\n",
+    "gdp = pd.Series([1., 1.21631, 1.4108], index=pd.Index(horizon, name='Time'))\n",
+    "demand_profile = gdp ** beta\n",
+    "demand_profile.plot(title='Demand')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "The `COMMODITY_BALANCE` equation ensures that `input`, `output`, and `demand` are met at each `level` in the energy system\n",
```

Comment:
Might also want to mention commodities here. Also note that `input` is the parameter, `input` times `ACT` is the amount of commodity created/produced/generated at that node. Better to make that distinction clear also for users working through the tutorial.

Reply:
done (these are comments are out of order for some reason, so I'm just adding a note that I've addressed it)